app/build.gradle

```diff
@@ -61,6 +61,7 @@ dependencies {
     compile 'com.github.bumptech.glide:glide:4.0.0-RC0'
     compile 'junit:junit:4.12'
     compile 'com.google.code.findbugs:annotations:2.0.1'
+    compile group: 'net.i2p.crypto', name: 'eddsa', version: '0.1.0'
```

Comment:
Can you please use the style that all other plugins are using?
`compile 'net.i2p.crypto:eddsa:0.1.0'`